integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -2659,6 +2666,50 @@ public static void createPVPVCAndVerify(V1PersistentVolume v1pv,
                 pvcName, namespace)));
   }
 
+  /**
```

Comment:
We have similar method ( createPvAndPvc) with same name in ItMonitoringExporter.java.  Can we merge these two.

Reply:
I need to fix MonExporter separately, it requires more changes for prometheus and grafana pvs I will do it in different jira